csharp/extractor/Semmle.Extraction/TrapExtensions.cs

```diff
@@ -19,6 +19,12 @@ public static void WriteLabel(this TextWriter trapFile, IEntity entity)
 
         public static void WriteSubId(this TextWriter trapFile, IEntity entity)
         {
+            if (entity is null)
+            {
+                trapFile.Write("<null>");
+                return;
```

Comment:
Shouldn't we log an extractor error in this case?

Reply:
There are two reasons for this. 1) We don't have an `Extractor` to log to in this case, and 2) We may as well change the specification of this method to handle `null` as well.